### PR TITLE
Import documentation

### PR DESCRIPTION
**Pull request by grobie**
In order to provide documentation for each individual version, this PR aims to move all Prometheus server specific documentation into the repository itself. This branch will be merged into the release-1.8 branch and should only contain Prometheus 1.8 documentation.

Links in documentation pages extracted from prometheus/docs need to be changed:
- for links to other Prometheus server specific pages, use normal markdown links (e.g. including `.md` file suffix)
- for links to pages on prometheus.io, use absolute URLs (e.g. starting with `https://prometheus.io`)

### CODE REVIEW

**Comment by fabxc** [comment]
👍 

What will we do about the current `documentation` folder? Just put it here and possibly allow `excludes` when importing to the website?

**Comment by grobie** [comment]
> Just put it here and possibly allow excludes when importing to the website?

Exactly that's the plan. I tried to do the exclude part, but it's tricky in nanoc3. Would be a lot easier in Nanoc4. There is an issue for that.

**Comment by grobie** [comment]
This is ready to get merged into `release-1.8`. It's more or less a plain copy of prometheus/docs pages into the `docs/` folder, just replacing links to be either absolute (including `https://prometheus.io`) or relative (with a `.md` suffix), and adding a basic index.md.

@brian-brazil @fabxc @juliusv

**Comment by juliusv** [comment]
:+1:

**Comment by grobie** [comment]
Happy to move /docs/alerting/rules/ here as well. Does it make sense to have that under `querying/` actually? Recording rules are there, but than neither creating recording nor alerting rules sounds like a querying concept to me. Maybe `configuration/{???.md,recording_rules.md,alerting_rules.md}`. Not sure how to rename the current configuration page though. Maybe just `configuration/configuration.md` ...

**Comment by juliusv** [comment]
`configuration/{configuration.md,recording_rules.md,alerting_rules.md}` sounds good to me.

**Comment by fabxc** [comment]
Sounds good 👍

**Comment by grobie** [comment]
@fabxc @juliusv pushed an update.

**Comment by fabxc** [comment]
The diff is not well reviewable but walking through the branch directly, everything looks good to me!

**Comment by juliusv** [comment]
Yep, :+1:

File: docs/installation.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+```bash
+docker build -t my-prometheus .
+docker run -p 9090:9090 my-prometheus
+```
+
+A more advanced option is to render the configuration dynamically on start
+with some tooling or even have a daemon update it periodically.
+
+## Using configuration management systems
```
Comment (brian-brazil) [inline docs/installation.md]: As a general note, I want to move these over to the Integrations page and put them with the coreos operator, as they're the same kind of thing.

**Comment by fabxc** [comment]
\o/ thank you!

**Pull request merged**